### PR TITLE
add exceptions to etherscan errors (blockchain-certificates#104)

### PR DESCRIPTION
**Pull request by akodate**
#104

### CODE REVIEW

File: cert_issuer/blockchain_handlers/ethereum/connectors.py [hunk 1]
```diff
@@ -60,13 +60,16 @@ class EtherscanBroadcaster(object):
...
+    def broadcast_tx(self, tx, api_token):        
         tx_hex = tx
 
         broadcast_url = self.base_url + '?module=proxy&action=eth_sendRawTransaction'
         if api_token:
             '&apikey=%s' % api_token
         response = requests.post(broadcast_url, data={'hex': tx_hex})
+        if 'error' in response.json():
+            logging.error("Etherscan returned an error: %s", response.json()['error'])
+            raise BroadcastError()
```
Comment (AnthonyRonning) [inline cert_issuer/blockchain_handlers/ethereum/connectors.py]: Could you have it broadcast `response.json()['error']` ?

Comment (akodate) [reply cert_issuer/blockchain_handlers/ethereum/connectors.py]: I've updated the PR. Does it match what you're looking for?

Comment (AnthonyRonning) [reply cert_issuer/blockchain_handlers/ethereum/connectors.py]: Yup, thanks!

**Comment by AnthonyRonning** [comment]
Just wondering, have you been able to test with a real error coming back from etherscan?

**Comment by akodate** [comment]
Yes, here's an example:
{'code': -32010, 'message': 'Transaction with the same hash was already imported.'}

**Comment by AnthonyRonning** [comment]
Sounds good, thank you for doing this!

**Comment by AnthonyRonning** [comment]
There's a failing test but it was introduced in a previous PR. I've reached out to that person to see if they could fix it, but won't stop your PR on it. Seems intermittent maybe.

**Comment by akodate** [comment]
Thanks, I’d love a solution to this error as well. In the meantime I’m thinking about just running delayed retries until it goes through.

**Pull request merged**